### DISCUSSION TITLE
How do I enforce HTTPS on my Coolify dashboard?

### DISCUSSION THREAD

**Discussion by dungsil** [coollabsio/coolify#1462] [Q&A] [answered]
Is there a way to support it?
Or do I need to configure a reverse proxy or something?

**Comment by andrasbacsai** [comment, marked as answer]
> Is there a way to support it? Or do I need to configure a reverse proxy or something?

You just need to set a fqdn starting with `https` in the `Settings` menu.
![image](https://github.com/coollabsio/coolify/assets/5845193/7ed9a32d-5ea6-4da6-90cf-07c5dfd65a92)

**Reply by loics2** [reply]
Is there some additional configuration to do? I have an SSL error (Error code: SSL_ERROR_RX_RECORD_TOO_LONG in Firefox) when I set the instance's domain to HTTPS and try to access it at `https://mydomain.tld:8000`.

I also use the same domain for an application deployed on my instance, could this be the issue?
